.github/workflows/lock.yml

```diff
@@ -0,0 +1,24 @@
+name: Lock closed issues and PRs after 30 days
+
+permissions:
+  issues: write
+  pull-requests: write
+
+on:
+  schedule:
+    - cron: "10 1 * * *"
+
+jobs:
+  lock:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/lock-threads@v3
+        with:
+          issue-comment: >
+            I'm going to lock this issue because it has been closed for 30 days. This helps our maintainers find and focus on the active issues.
+            If you've found a problem that seems similar to this, please open a new issue and complete the issue template so we can capture all the details necessary to investigate further.
+          issue-inactive-days: 30
+          pr-comment: >
+            I'm going to lock this pull request because it has been closed for 30 days. This helps our maintainers find and focus on the active issues.
```

Comment:
Nit: Still references issues here (and in the following line). Maybe change it to use 'pull requests' instead.

Reply:
That being said, I wonder if opening issues still makes more sense than open a new pull request. Feel free to ignore.